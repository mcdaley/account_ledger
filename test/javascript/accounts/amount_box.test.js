//-----------------------------------------------------------------------------
// test/javascripts/accounts/amount_box.test.js
//-----------------------------------------------------------------------------
import React      from 'react'
import AmountBox  from '../../../app/javascript/components/accounts/amount_box'
import renderer   from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer
    .create(<AmountBox type="credit" text="Credits" total="55.00" />)
    .toJSON()
  expect(tree).toMatchSnapshot()
});