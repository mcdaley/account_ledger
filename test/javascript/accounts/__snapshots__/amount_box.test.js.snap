// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="card"
>
  <div
    className="card-header"
  >
    Credits
  </div>
  <div
    className="card-body account-card-body"
  >
    <h5
      className="card-title account-card-title"
    >
      $55.00
    </h5>
  </div>
</div>
`;
